bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ZeebeEndEventValidationTest.java

```diff
@@ -116,9 +115,6 @@ private static Stream<EndEventTypeBuilder> supportedEndEventTypes() {
 
   private static Stream<EndEventTypeBuilder> unsupportedEndEventTypes() {
     return Stream.of(
-        new EndEventTypeBuilder(
-            CompensateEventDefinition.class,
-            endEvent -> endEvent.compensateEventDefinition().compensateEventDefinitionDone()),
```

Comment:
:wrench: We should add compensation to the supported end event types.

Reply:
Do you mean this [line](https://github.com/camunda/zeebe/blob/884b807e7edc76c1e03cbd664223c2544920a996/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/EndEventValidator.java#L43) ?

Reply:
Oh 😆